code/__HELPERS/names.dm

```diff
@@ -168,7 +168,7 @@ GLOBAL_DATUM(syndicate_code_response_regex, /regex)
 	var/threats = strings(ION_FILE, "ionthreats")
 	var/foods = strings(ION_FILE, "ionfood")
 	var/drinks = strings(ION_FILE, "iondrinks")
-	var/list/locations = GLOB.teleportlocs.len ? GLOB.teleportlocs : drinks //if null, defaults to drinks instead.
+	var/locations = strings(LOCATIONS_FILE, "locations")

```

Comment:
Is it not reasonable to just make the list get generated sooner? Why the workaround that will need to be updated manually?

Reply:
I thought about some solutions to this, but I ended up with a file so I could also clean the list. There are a fair few area names that just dont work all that well as codewords, because noone would say the exact names in natural conversation. Like Security outpost - research division and so on.

Reply:
Fair enough